docs/containers-transports.5.md

```diff
@@ -43,6 +43,7 @@ The containers-registries.conf(5) further allows for configuring various setting
 Note that a _docker-reference_ has the following format: `name[:tag|@digest]`.
 While the docker transport does not support both a tag and a digest at the same time some formats like containers-storage do.
 Digests can also be used in an image destination as long as the manifest matches the provided digest.
+The docker transport supports pushing images without a tag or digest to a registry when the image name is suffixed with `@@unknown-digest@@`. Images without a tag or digest can be used to create manifest lists after the images have been pushed.
 The digest of images can be explored with skopeo-inspect(1).
 If `name` does not contain a slash, it is treated as `docker.io/library/name`.
```

Comment:
This is getting long and hard to parse. A paragraph break above this line?

Reply:
Fixed